resources/terminology/ContactPointSystem-zib-ContactInformation-TelephoneNumbers.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="ContactPointSystem-zib-ContactInformation-TelephoneNumbers"/>
+    <url value="http://nictiz.nl/fhir/ValueSet/ContactPointSystem-zib-ContactInformation-TelephoneNumbers"/>
+    <version value="4.0.1"/>
+    <name value="contactPointSystemZibContactInformationTelephoneNumbers"/>
+    <title value="ContactPointSystem for zib ContactInformation TelephoneNumbers"/>
+    <status value="active"/>
+    <experimental value="false"/>
+    <publisher value="Nictiz"/>
+    <description value="Telecommunications form for contact point, restricted to the use for the TelephoneNumbers concept of zib ContactInformation."/>
```

Comment:
Form?

Miss hetzelfde houden als emails:
Use of contact point, restricted to the use for the TelephoneNumbers concept of zib ContactInformation.

Reply:
"form" is overgenomen van de omschrijving van het CodeSysteem: "Telecommunications form for contact point."

Reply:
Aha, het komt hier uit: http://hl7.org/fhir/valueset-contact-point-system.html
Ok! Laten we het inderdaad zo laten.